screwdriver/screwdriver.go

```diff
@@ -127,6 +126,9 @@ func (sd *sdAPI) jwt() (string, error) {
 
 	tokenResponse := new(tokenResponse)
 	err = json.NewDecoder(res.Body).Decode(tokenResponse)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse JWT response: %v", err)
+	}
 
 	return tokenResponse.JWT, err
```

Comment:
`nil` is better like this?
```suggestion
	return tokenResponse.JWT, nil
```